src/main/java/com/minecolonies/coremod/colony/buildings/registry/BuildingRegistry.java

```diff
@@ -239,6 +240,7 @@ public static AbstractBuilding create(final Colony colony, @NotNull final TileEn
 
         if (building != null && parent.getWorld() != null && !(building instanceof PostBox))
         {
+            building.setRotation(BlockUtils.getRotationFromFacing(parent.getWorld().getBlockState(parent.getPosition()).getValue(AbstractBlockHut.FACING)));
```

Comment:
This here is the default rotation, for when placed by hand, the real rotation is set later then, if existent